public/components/app.vue

```diff
@@ -347,7 +373,7 @@
                             .replace(/[^0-9a-z:\.]/ig, ''),
                         recordsLimit = parseInt(this.recordsLimit, 10);
 
-                    parts['redirect_to'] = redirectIp.length >= 4 ? redirectIp : null;
+                    parts['redirect_to'] = redirectIp.match(ip({exact: true})) !== null ? redirectIp : null;
```

Comment:
May be better to extend `this.redirectIP` to saving validation status? It't make redundant double validation.

Reply:
yep, it seems like a pretty good idea!